ENGyn.NodesTestPlatform/ENGyn.NodesTestPlatform/Services/IConfigurationService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.IO;
+
+namespace ENGyn.NodesTestPlatform.Services
+{
+    public interface IConfigurationService
+    {
+        DirectoryInfo createsProjectDirectory(string directoryName);
```

Comment:
Change `createsProjectDirectory` to `CreatesProjectDirectory` (PascalCase)

Reply:
DONE